native/android/app/src/main/java/com/native/MainActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.skyscanner;
```

Comment:
Quick note - as the folder the app lives in is called `native`, this was originally `com.native`. However this wouldn't build because it's apparently an invalid package name, so I brute forced my way out of it by changing it to `com.skyscanner` everywhere it occurred.

Reply:
perhaps `com.backpack`?

Reply:
Can do that, sure!

Reply:
Done